docker/fdb_tests/scripts/tests/full_backup_test.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+set -e -x
+
+export WALG_STREAM_CREATE_COMMAND='TMP_DIR=$(mktemp -d) && chmod 777 $TMP_DIR && fdbbackup start -d file://$TMP_DIR -w 1>&2 && tar -c -C $TMP_DIR .'
+export WALG_STREAM_RESTORE_COMMAND='TMP_DIR=$(mktemp -d) && chmod 777 $TMP_DIR && tar -xf - -C $TMP_DIR && BACKUP_DIR=$(find $TMP_DIR -mindepth 1 -print -quit) && fdbrestore start -r file://$BACKUP_DIR -w --dest_cluster_file "/var/fdb/fdb.cluster"  1>&2'
+export WALG_FILE_PREFIX='/tmp/wal-g'
+
+fdbcli -C /var/fdb/fdb.cluster --exec 'configure new single memory; writemode on; set test_key test_value'
```

Comment:
It would be better to use something like pg_bench(PostgreSQL) or sysbench(MySQL) here.
It's possible to write a script that imitates lots of writes and reads simultaneously.

Reply:
And some DDL queries